src/fu-quirks.h

```diff
@@ -131,6 +129,32 @@ const gchar	*fu_quirks_lookup_by_udev_device	(FuQuirks	*self,
  */
 #define	FU_QUIRKS_GUID				"Guid"
 
+/**
+ * FU_QUIRKS_PARENT_GUID:
+ * @key: the USB device ID, e.g. `USB\VID_0763&PID_2806`
```

Comment:
All these instances of "USB device ID" in the new quirks should actually be FuDeviceId shouldn't they be?

Reply:
You're right, they should all be `DeviceInstanceID=USB\VID_0763&PID_2806`. Will fix.